src/styles/_mixins.scss

```
@@ -0,0 +1,14 @@
+@mixin resets() {
```

```suggestion
@mixin resets {
```
якщо ти в mixin не передаєш праметрів то пиши їх без дужок)

---

Виправила